lib/components/ArtistScreen/artist_screen_content.dart

```diff
@@ -50,24 +64,56 @@ class _ArtistScreenContentState extends State<ArtistScreenContent> {
 
           return Scrollbar(
               child: CustomScrollView(slivers: <Widget>[
-            const SliverPadding(padding: EdgeInsets.fromLTRB(0, 15.0, 0, 0)),
-            SliverToBoxAdapter(
-                child: Text(
-              AppLocalizations.of(context)!.topSongs,
-              style: const TextStyle(fontSize: 18, fontWeight: FontWeight.bold),
-            )),
+            SliverAppBar(
+              title: Text(widget.parent.name ??
+                  AppLocalizations.of(context)!.unknownName),
+              // 125 + 146 is the total height of the widget we use as a
+              // FlexibleSpaceBar. We add the toolbar height since the widget
+              // should appear below the appbar.
+              // TODO: This height is affected by platform density.
+              expandedHeight: kToolbarHeight + 125 + 146,
```

Comment:
What was the initial way to calculate the 186?

Reply:
No idea. There was some initial value, and every time I modified the height of the widgets I updated the offset 🙃